app/javascript/components/modals/PublishSolutionModal.tsx

```diff
@@ -49,10 +49,18 @@ export const PublishSolutionModal = ({
   )
 
   return (
-    <Modal {...props} className="m-change-published-iteration">
+    <Modal
+      {...props}
+      aria={{
+        modal: true,
+        labelledby: 'publish-solution-label',
+        describedby: 'publish-solution-description',
+      }}
+      className="m-change-published-iteration"
+    >
       <form data-turbo="false" onSubmit={handleSubmit}>
-        <h3>Publish your solution</h3>
-        <p>
+        <h3 id="publish-solution-label">Publish your solution</h3>
+        <p id="publish-solution-description">
```

Comment:
sure, I think these could be automatically generated by a function, because we need these on all modals, and the pattern is very simple. `aria-{hex}-{modal-name}-[description|label]`. I'll play with this tomorrow.

Reply:
OK great :)

We can always get this green with this, and then have a second PR that adds them everywhere using a function?